clinicaltrials/frontend/management/commands/load_data.py

```diff
@@ -204,18 +213,448 @@ def process_data():
+                and is_covered_phase(td["phase"])
+                and is_not_device_feasibility(td["primary_purpose"])
+                and td["start_date"] < effective_date
+                and td["available_completion_date"] >= effective_date
+                and is_not_withdrawn(td["study_status"])
+            ):
+                new_pact_flag = True
+            else:
+                new_pact_flag = False
+
+            if old_pact_flag == True or new_pact_flag == True:
+                td["included_pact_flag"] = True
+            else:
+                td["included_pact_flag"] = False
+
+            td["location"] = dict_or_none(parsed_json, [cs, "location_countries"])
+
+            td["has_results"] = does_it_exist(soup.results_first_submitted)
+
+            td["pending_results"] = does_it_exist(soup.pending_results)
+
+            td["pending_data"] = dict_or_none(parsed_json, [cs, "pending_results"])
+
+            if (
+                (td["act_flag"] == True or td["included_pact_flag"] == True)
+                and date.today()
+                > td["available_completion_date"]
+                + relativedelta(years=1)
+                + timedelta(days=30)
+                and (
+                    td["has_certificate"] == 0
+                    or (
+                        date.today()
+                        > td["available_completion_date"]
+                        + relativedelta(years=3)
+                        + timedelta(days=30)
+                    )
+                )
+            ):
+                td["results_due"] = True
+            else:
+                td["results_due"] = False
+
+            td["results_submitted_date"] = (
+                str_to_date(soup.results_first_submitted)
+            )[0]
+
+            td["last_updated_date"] = (str_to_date(soup.last_update_submitted))[0]
+
+            td["certificate_date"] = (
+                str_to_date(soup.disposition_first_submitted)
+            )[0]
+
+            td["enrollment"] = t(soup.enrollment)
+
+            td["sponsor"] = t(soup.sponsors.lead_sponsor.agency)
+
+            td["sponsor_type"] = t(soup.sponsors.lead_sponsor.agency_class)
+
+            td["collaborators"] = dict_or_none(
+                parsed_json, [cs, "sponsors", "collaborator"]
+            )
+
+            td["exported"] = t(soup.oversight_info.is_us_export)
+
+            td["url"] = t(soup.url)
+
+            td["official_title"] = t(soup.official_title)
+
+            td["brief_title"] = t(soup.brief_title)
+
+            td["title"] = td["official_title"] or td["brief_title"]
+
+            if td["official_title"] is not None:
+                td["title"] = td["official_title"]
+            elif td["official_title"] is None and td["brief_title"] is not None:
+                td["title"] = td["brief_title"]
+            else:
+                td["title"] = None
+
+            # add this to "if" statement after testing: 'and completion_date is not null')
```

Comment:
@NickCEBM do you want to do this?

Reply:
Have we confirmed that this code creates an identical CSV to the SQL yet?

Reply:
Assume we have so added (knowing we can go back if not).